src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/RazorLanguageServerCustomMessageTargets.cs

```diff
@@ -26,6 +27,7 @@ internal static class RazorLanguageServerCustomMessageTargets
     public const string RazorFoldingRangeEndpoint = "razor/foldingRange";
     public const string RazorHtmlFormattingEndpoint = "razor/htmlFormatting";
     public const string RazorHtmlOnTypeFormattingEndpoint = "razor/htmlOnTypeFormatting";
+    public const string RazorDocumentSymbolEndpoint = "razor/documentSymbol";
```

Comment:
This is in the wrong spot. "Cross platform" here means "We support this endpoint in VS and VS Code", I just couldn't be explicit about it at the time I wrote the comment, because the new extension was a secret. This should go in "Still to migrate" I guess.

Reply:
I thought document symbol would be supported in both?

Reply:
Yes, sorry, my bad working again. VS Code support document symbol (I suppose?), but we don't have a handler for this endpoint in the C# Extension for VS Code :)

Reply:
ah gotcha, I'll move